glean-core/android/src/main/java/mozilla/telemetry/glean/utils/JSONObject.kt

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.utils
+
+import org.json.JSONObject
+import java.util.TreeMap
+
+/**
+ * Returns the value mapped by {@code key} if it exists, and
+ * if the value returned is not null. If it's null, it returns null
+ */
+fun JSONObject.tryGet(key: String): Any? = if (isNull(key)) null else get(key)
```

Comment:
Are we using anything in this file at all?

Reply:
`tryGetLong()` is the only thing used.  I've combined the functions we use into a single file and test file so it will be easier to review once I commit this change.